packages/bpk-theming/README.md

```diff
@@ -35,8 +35,9 @@ export default class App extends Component {
 
 ## Props
 
-| Property            | PropType        | Required | Default Value |
-| -----------         | --------------- | -------- | ------------- |
-| children            | node            | true     | -             |
-| theme               | object          | false    | null          |
-| themeAttributes     | arrayOf(string) | true     | -             |
+| Property            | PropType                           | Required         | Default Value |
+| -----------         | ---------------------------------- | ---------------- | ------------- |
+| children            | node                               | true             | -             |
+| themeAttributes     | arrayOf(string)                    | true             | -             |
+| component           | oneOf(function, string)            | false            | div           |
```

Comment:
shouldn't this be `Element`?

Reply:
No because `Element` means an already rendered/instantiated component. E.g. `<Component />` vs `Component`

Reply:
Cool. Out of interest how would you flow type this? :P